eiffel-syntax-and-usage/glossary.md

```diff
@@ -0,0 +1,65 @@
+<!---
+   Copyright 2020 Ericsson AB.
+   For a full list of individual contributors, please see the commit history.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+--->
+
+# Eiffel Glossary
+This a non-exhaustive, alphabetical list of terms used in the Eiffel Vocabulary and its documentation.
+
+ - [Terms Used in Event Definitions](#terms-used-in-event-definitions)
+   - [Activity](#activity)
+   - [Artifact](#artifact)
+   - [Environment](#environment)
+   - [Event](#event)
+   - [Source Change](#source-change)
+   - [Submit](#submit)
+
+ - [Terms Used in Documentation](#terms-used-in-documentation)
+   - [Pipeline](#pipeline)
+   - [Step](#step)
+
+## Terms Used in Event Definitions
+These terms are either part of the event names in the Eiffel Vocabulary, or they are part of parameter names or values in the events. Users of Eiffel should be able to rely on these terms to be kept, unless a major change to event types are done.
+
+### Activity
+An _activity_ is any kind of action in a CI/CD system, normally triggered by an operation done in an SCM system or by a previous activity. It's purpose is to notify what is going on and what has finished, and when, and with what outcome.
```

Comment:
"It's purpose is to notify" - What is "_it's_ " in this sentence? Are we referring to the Activity** _events_ here? 

Perhaps phrase it: "The Eiffel protocol has events related to activities, to describe what is going on and what has finished, and when, and with what outcome"? 

Can we include TestCaseTriggered/Started/Finished to this term? 
E.g.: "Some events are specific, like TestCase** events, while others are more generic: such as Activity** events."

Those we mention should perhaps also have linkts to further reading.

Reply:
"what is going on, what has finished, when did it happen and with what outcome"

Reply:
Agreed

Reply:
Now the text has changed but it was also misspelled. It's supposed to be 'Its'. =S

Reply:
That version of the code is also outdated now :) It's rephrased again.